src/dialog/tests/unit/Dialog.ts

```diff
@@ -32,7 +32,11 @@ const expectedCloseButton = function() {
 };
 
 const expected = function(widget: Harness<Dialog>, open = false, closeable = false, children: any[] = []) {
-	return v('div', { classes: css.root }, open ? [
+	return v('div', {
+		classes: css.root,
+		dir: null,
```

Comment:
Are these added by the `I18NMixin`?

Reply:
Yes, the `I18nMixin` adds both `lang` and `dir` attributes.